### ISSUE TITLE
TangleID: TangleID Core monorepo

### ISSUE THREAD

**Issue by ender503** [DLTcollab/general#121]
We need to create a single repository to maintain multiple NPM packages, otherwise we had to maintain these NPM package in each repository.

**Comment by ender503** [comment]
I create a new repository in [TangleID/tangleid-core: TangleID JavaScript monorepo](https://github.com/TangleID/tangleid-core).

Since we had this repo, the `tangleid-connect` and `tangleid-did` are archived.

**Comment by jserv** [comment]
Will `SemVer` be applied to `tangleid-core`? If so, how can we manage these versions?

**Comment by ender503** [comment]
Yes, we will use `SemVer`, just like [facebook/react](https://github.com/facebook/react/releases).

And I will write the tutorial for managing the version with `lerna`.
